### ISSUE TITLE
Decrease docs website file size

### ISSUE THREAD

**Issue by JKillian** [palantir/blueprint#39]
I was showing the BP website to a person unfamiliar with it, and (about) 10MB of content ends up being a really heavy initial load. It would be great to reduce this, also I think in general the developer community prefers less-heavy experiences.

Possible ideas:
* Remove all angular examples from public docs site / move them to a separate page
* Make sure that all sources (JS/CSS) are appropriately minified
* Make sure that no duplicate dependencies are getting included
* Eliminate any extra dependencies that are only being used for a small feature. Or, make sure to use Webpack 2's tree-shaking feature appropriately.
* Make sure inline source maps aren't included anywhere (#112)

**Comment by liquidox** [comment]
Yes, you should have fixed this before shipping.

**Comment by llorca** [comment]
We fixed #112 already. The Chrome console shows 8 requests and 894KB downloaded (half of that after caching):
![image](https://cloud.githubusercontent.com/assets/199754/22036315/e827f268-dca7-11e6-8d94-c51e66f68fed.png)

Still more work to do of course, we'll tackle it as part of version 2.0 when we revamp the docs to include upcoming new projects, in addition to a new landing page which will be more lightweight. Gonna close this one as we've made good progress and next efforts will be focused on 2.0.
